### PR TITLE
Remove debug lines

### PR DESCRIPTION
**Pull request by Tensuko**
Those debug lines are not needed anymore.
Remove the option to toggle it on the HUD.

### CODE REVIEW

File: settings.lua [hunk 1]
```diff
@@ -3965,23 +3965,17 @@ end
 
 --- @class CourseDrawModeSetting : SettingList
 CourseDrawModeSetting = CpObject(SettingList)
--- 2D/debug lines display options
+-- 2D lines display options
 CourseDrawModeSetting.COURSE_2D_DISPLAY_OFF	 = 0;
 CourseDrawModeSetting.COURSE_2D_DISPLAY_2DONLY	 = 1;
-CourseDrawModeSetting.COURSE_2D_DISPLAY_DBGONLY = 2;
```
Comment (pvaiko) [inline settings.lua:3971]: should check if these constants are used elsewhere (yes, they are...)

**Review by pvaiko — CHANGES_REQUESTED** [review]

File: settings.lua [hunk 2]
```diff
@@ -838,7 +838,7 @@ function courseplay:showFieldEdgePath(vehicle, pathType)
 end;
 
 function courseplay:changeDrawCourseMode(vehicle, changeBy)
-	vehicle.cp.drawCourseMode = courseplay:varLoop(vehicle.cp.drawCourseMode, changeBy, courseplay.COURSE_2D_DISPLAY_BOTH, courseplay.COURSE_2D_DISPLAY_OFF);
+	vehicle.cp.drawCourseMode = courseplay:varLoop(vehicle.cp.drawCourseMode, changeBy, courseplay.COURSE_2D_DISPLAY_OFF);
```
Comment (pvaiko) [inline settings.lua:841]:
```lua
vehicle.cp.drawCourseMode = courseplay:varLoop(vehicle.cp.drawCourseMode, changeBy, CourseDrawModeSetting.COURSE_2D_DISPLAY_2DONLY, courseplay.COURSE_2D_DISPLAY_OFF);
```

File: settings.lua [hunk 3]
```diff
@@ -838,8 +838,7 @@ function courseplay:showFieldEdgePath(vehicle, pathType)
 end;
 
 function courseplay:changeDrawCourseMode(vehicle, changeBy)
-	vehicle.cp.drawCourseMode = courseplay:varLoop(vehicle.cp.drawCourseMode, changeBy, courseplay.COURSE_2D_DISPLAY_BOTH, courseplay.COURSE_2D_DISPLAY_OFF);
-	vehicle.cp.hud.changeDrawCourseModeButton:setActive(vehicle.cp.drawCourseMode ~= courseplay.COURSE_2D_DISPLAY_OFF);
```
Comment (pvaiko) [inline settings.lua:842]: why did you remove this line?

Comment (schwiti6190) [reply settings.lua:842]: The complete function can be removed, as this is old code before the CourseDrawModeSetting was implemented and no longer used.
